pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/ast/StmtForStaticSqlLoop.java

```diff
@@ -31,11 +31,34 @@
 package com.cubrid.plcsql.compiler.ast;
 
 import com.cubrid.plcsql.compiler.StaticSql;
+import com.cubrid.plcsql.compiler.ast.loopOpt.SqlUse;
 import com.cubrid.plcsql.compiler.visitor.AstVisitor;
 import java.util.ArrayList;
 import org.antlr.v4.runtime.ParserRuleContext;
 
-public class StmtForStaticSqlLoop extends StmtForSqlLoop {
+public class StmtForStaticSqlLoop extends StmtForSqlLoop implements SqlUse {
+
+    public StmtLoop containerLoop;
```

Comment:
```suggestion
    private StmtLoop containerLoop;
```
setter가 존재하므로 getter도 추가하여, public 보다는 private으로 선언하여 관리를 용이하게 하는 것이 좋을 것 같습니다.